com.fugro.roames.displaz.json

```diff
@@ -0,0 +1,54 @@
+{
+  "id": "com.fugro.roames.displaz",
+  "branch": "master",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.11",
+  "sdk": "org.kde.Sdk",
+  "command": "displaz",
+  "tags": [
+    "master"
+  ],
+  "finish-args": [
+    "--socket=x11",
+    "--socket=wayland",
+    "--device=dri"
+  ],
+  "modules": [
+    {
+      "name": "displaz",
+      "buildsystem": "simple",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/nigels-com/displaz.git",
+          "branch": "flatpak"
```

Comment:
You should specify the `commit` as well.

See also:
https://github.com/flathub/flathub/wiki/App-Requirements#stable-releases-reproducible-builds